echo2.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "common/common/assert.h"
 
+namespace Lyft {
```

Comment:
So what is the plan here? Are we somehow going to automate putting all code in the repo in a namespace? Yeah I agree putting it in Envoy namespace seems better.